dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/StringConcatenateAssignmentNode.java

```diff
@@ -10,12 +10,13 @@
 import java.util.Objects;
 
 /**
- * A node for the string concatenation compound assignment:
- *
- * <pre>
+ * @deprecated Use {@code #AssignmentNode()} instead. A node for the string concatenation compound
```

Comment:
The `@deprecated` part is the last thing in the javadoc block, separate from the rest.

Reply:
fixed.

Reply:
fixed.